### DISCUSSION TITLE
Consider adding WoA Dialer and Chat

### DISCUSSION THREAD

**Discussion by upintheairsheep** [IviriusCommunity/Rebound#100] [Ideas]
https://github.com/WOA-Project/woadialer/ https://github.com/WOA-Project/Chat - Dialer and SMS apps respectively for LTE Windows devices, originally intended for smartphones that have been ported to Windows 10/11, consider adding if it works on LTE laptops (we need someone with an LTE laptop to test).

**Comment by Lamparter** [comment]
I'm all for this, @IviriusMain what do you think?

**Reply by IviriusMain** [reply]
FD1, doesn't match the main criteria for an app to become part of Rebound.
